src/components/ProductCard.js

```diff
@@ -121,10 +123,14 @@ const ProductCard = ({
   gitHubRepo,
   subjects,
 }) => {
+  const intl = useIntl()
   return (
     <Card>
       <ImageWrapper background={background}>
-        <Image fixed={image} alt={`${name} logo`} />
+        <Image
+          fixed={image}
+          alt={`${name} ${translateMessageId("logo", intl)}`}
```

Comment:
Something I've realised is that by just allowing a translator to translate "logo", these alt tags may not make sense in some languages.

Although Solidity logo in English... in Spanish "Solidity" would come after logo like "Logotipo de Solidity" (that's just a poor google translate).

I think we should use `alt={alt}` in this component and then wherever ProductCard is used we add description to the array and reference a full key in the JSON file, e.g. `"solidity-logo": "Solidity logo"`

I've done it for the stablecoins page here, for reference: https://github.com/ethereum/ethereum-org-website/pull/1938/commits/842088053f17f65d199ba884741d3cdc2c02811b

Reply:
Agreed! Nice catch. Same with my comment as well, I think where possible we should keep components dumb & explicitly pass the alt text as a prop.